ztools/x509/cert_pool.go

```diff
@@ -77,14 +77,13 @@ func (s *CertPool) AddCert(cert *Certificate) {
 }
 
 // AppendCertsFromPEM attempts to parse a series of PEM encoded certificates.
-// It appends any certificates found to s and returns true if ALL certificates
```

Comment:
We actually made this change ourselves, to make the function be more strict when loading in a Root CA file.